vote_text_drafts/rpip-28.md

```diff
@@ -0,0 +1,29 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# Deposits Under the Minimum (RPIP-28)
+
+### Outcomes
+ If this vote passes, it will be possible to make a minipool deposit when under the "minimum" RPL threshold by depositing the ETH and staking (at least) the minimum amount of RPL needed for a single minipool of that type (this is in _addition_ to the current method). Note that this requires a smart contract change, so it won't take effect until it is included in a larger smart contract release.
+
+### Rationale
+It is currently possible to create a new node to deposit a minipool. In other words, this feature does _not_ change the ability for a Node Operator to create a new minipool - they'll be able to create one for the same amount of ETH and RPL. What it _will_ impact is the convenience. A handful of folks have said they would have created more minipool(s) if they could've added them to their existing node while being under the minimum. This would help us provide more rETH supply, while not trading off much of anything.
```

Comment:
```suggestion
It is currently possible to create a new node to deposit a minipool, so this feature does _not_ change the ability for a Node Operator to create a new minipool. They'll be able to create one for the same amount of ETH and RPL. 

This change instead makes the process more convenient. A handful of folks have said they would have created more minipool(s) if they could've added them to their existing node while being under the minimum.

 This should help us provide more rETH supply, with no significant trade-off. 
```

Mostly fiddling tbh.

Reply:
I do like the additional whitespace. Ended somewhere in between